test/e2e/app-dir/empty-static-params/empty-static-params.test.ts

```diff
@@ -0,0 +1,12 @@
+import { createNextDescribe } from 'e2e-utils'
+
+createNextDescribe(
+  'empty-static-params',
+  {
+    files: __dirname,
+  },
+  () => {
+    // Runs a dummy test to ensure the test suite has built while working around the fact that `next start` does not work with exports
+    it('should have built', async () => {})
+  }
+)
```

Comment:
This is janky, but the comment should give all required info. Totally open to any better avenue here, either with integration tests or a modification to the e2e test runner to allow static export testing.